test/integration/scheduler/scheduler_test.go

```diff
@@ -285,6 +257,7 @@ var _ = ginkgo.Describe("Scheduler", func() {
 
 	ginkgo.It("Should schedule jobs from the selected namespaces", func() {
 		clusterQ := testing.MakeClusterQueue("cluster-queue-with-selector").
+			QueueingStrategy(kueue.StrictFIFO).
```

Comment:
I think we may also need to watch the events of `ns` to trigger the movement of workloads in ClusterQueue with BestFIFO.

Reply:
yes, can you please open a tracking issue for that? I am guessing this test didn't pass without changing the strategy because of this, right?

Reply:
Yes opened https://github.com/kubernetes-sigs/kueue/issues/234

Reply:
thanks, this is actually a bug that we may need to backport to 0.1.0